week2/help.txt

```diff
@@ -1,25 +1,27 @@
-This is the help page for CLI ToDo app
 
-Instructions:
+This is the help page for CLI ToDo app
```

Comment:
Nice full help file with examples! This is the kind of thing users like to see, and makes them feel better about your program.